README.md

```diff
@@ -1,11 +1,10 @@
 * NeXus: https://www.nexusformat.org/
 * Citation: [![Zenodo DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1472392.svg)](https://doi.org/10.5281/zenodo.1472392)
-* Documentation: https://manual.nexusformat.org/
+* Documentation: https://nexusformat.github.io/definitions
```

Comment:
This is not needed.  Let's leave it *branded* to nexusformat.org.

Reply:
Ah ok, I misunderstood your comment. I put it back to the original URL.